docusaurus/docs/Android/04-compose/02-component-architecture.mdx

```diff
@@ -79,6 +81,7 @@ override fun onCreate(savedInstanceState: Bundle?) {
     setContent {
         ChatTheme { // Theme wrapper
             ChannelList(
+                modifier = Modifier.fillMaxSize(),
                 viewModel = channelListViewModel,
                 onChannelClick = ::openMessages,
```

Comment:
I think `::openMessages` is a bit misleading here as the docs don't show this method, so, in my opinion better to do:
```
onChannelClick = {
    // Open messages screen
}
```

Reply:
Updatged!